config/controller.yaml

```diff
@@ -57,6 +57,10 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
+        securityContext:
+          runAsUser: 1337
```

Comment:
Can you add a blurb in the documentation that we run the components with non-root user 1337? I think https://github.com/knative/serving/blob/master/docs/resources-overview.md#components seems reasonable.

Reply:
For sure, good thought.